data/settings.js

```diff
@@ -1620,6 +1660,46 @@ plugin {
 \`\`\``
 	},
 
+	acl_ignore: {
+		added: {
+			settings_acl_global_settings_added: false,
+		},
```

Comment:
I was also wondering about this. But from the other perspective: How to retain the "old" documentation for people who are still on 2.3.x? I wonder whether we should have even more updates?

Otherwise how does the user know whether a setting is only valid for 2.4/3.0 or also works in 2.3.x? (Without `doveconf` failing.)

Reply:
I'll inquire Michael about this. He might have a more definitive opinion on the topic.

Reply:
Haven't had chance to review full MR series yet, but idea:
- Settings entry lists "old" name and "new" name
- We create a single change entry for all config changes that land in a single release (although that becomes a bit more challenging if the Pro and CE landings are different.)
- If a setting has an "old" name entry, we display the changed badge.  Doesn't have to be same as the other "Change" badges, we could create a specific one for settings changes.

e.g., in settings.js, for any setting we can have:

```
setting_name: {
   ...
   old_config: {
       name: 'old_name',
       changed: 'config_rewrite_batch_1',
       // removed: 'config_rewrite_batch_x'
    },
   ...
},
```

And then define a single 'config_rewrite_batch_1' in updates.js.  Removed will be populated in the future when the old version no longer works.  Then it's just some templating magic needed to display this nicely on the settings pages.

Reply:
I would suggest we postpone this discussion in another PR. This endeavor was intended to port the old config-rewrite changes onto the new system, and I hope it does this. I agree that there are more unoptimal solutions and/or missing items still obvious. And I think this is better suited as a separate work item.

Reply:
@slusarz I think this is your action item: Create a JIRA task to track consolidated setting updates for config-rewrite vs. main.